src/snowflake/snowpark/_internal/type_utils.py

```diff
@@ -698,33 +699,44 @@ def python_type_to_snow_type(
             return StructType(), False
         return MapType(key_type, value_type), False
 
-    if installed_pandas:
-        pandas_series_tps = [PandasSeries, pandas.Series]
-        if tp in pandas_series_tps or (tp_origin and tp_origin in pandas_series_tps):
-            return (
-                PandasSeriesType(
-                    python_type_to_snow_type(tp_args[0], is_return_type_of_sproc)[0]
-                    if tp_args
-                    else None
-                ),
-                False,
-            )
+    if get_installed_pandas():
+        pandas_types = get_pandas_types()
+        pandas = get_pandas()
+        if pandas_types:
```

Comment:
I don't think we need this check, or the check on line 702 can be removed
because get_pandas_types also checks `if get_installed_pandas():`